test/__snapshots__/index.test.js.snap

```diff
@@ -1,5 +1,15 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`handleHideAll should hide all other open elements, and toggle the element clicked 1`] = `
+"
+            <div data-toggle-name=\\"one\\" class=\\"\\" data-toggle-group=\\"groupName\\"></div>
+            <div data-toggle-name=\\"two\\" data-toggle-group=\\"groupName\\" class=\\"is-hidden\\"></div>
```

Comment:
Would this not do the same if all of the data-toggle-name attribute's were named "one"?

Also, once happy with approach, will need to add docs for the functionality into the README as well.

Reply:
The payment page has dynamic section names and multiple toggle buttons, so each section would need a unique toggle-name.